Luck | 2021-06-22 03:05:17 UTC | #1

ic-rosetta-api 0.8.0
Error in syncing blocks:InternalError(false,Some({"error_message":String("File exists (os error 17)")}))
What should I do?

-------------------------

